nexus/reconfigurator/planning/src/blueprint_editor/sled_editor/zones.rs

```diff
@@ -153,29 +149,12 @@ impl ZonesEditor {
     }
 }
 
-impl TryFrom<BlueprintZonesConfig> for ZonesEditor {
```

Comment:
I was happy to see this go away :)

Reply:
Yeah, getting rid of this is excellent. :+1: